### PR TITLE
Feature/#142313879/additional funtionalities

### PR DESCRIPTION
**Pull request by freemile2ru**
## What does this PR do?
Add game victory tweet functionality
Add random charity quotes display functionality

## Description of Task to be completed?
this PR includes the tweet game victory functionality and random quotes display on game end feature

## How should this be manually tested?
this can be tested by playing the game on the review app created from this PR

## Any background context you want to provide?
N/A

## What are the relevant pivotal tracker stories?
Additional functionalities
**Description:**
Tweet you game victory when you when a round this draw more awareness about the game
Add charity quotes to end of the game display screen

## Screenshots (if appropriate)
screen shot 2017-03-23 at 4 09 59 pm
screen shot 2017-03-23 at 4 09 26 pm

## Questions:
N/A

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ gulp.task('nodemon', () => {
 gulp.task('server', ['nodemon'], () => {
     browserSync.init({
         proxy: "http://localhost:3000",
-        port: 7000,
+        port: 7001,
```
Comment (ooduye) [inline gulpfile.js]: Why did you change to port 7001?

Comment (freemile2ru) [reply gulpfile.js]: hello @andela-ooduye thats a very good question. local host 7000 is not functional on my laptop for some weird reasons  so i had to change it on other to be able to test the app on my system. but i have reverted this now
